build.ps1

```diff
@@ -68,15 +68,15 @@ Function BuildProject($path, $message) {
     }
 }
 
-Function RunApp($path, $message, $args) {
+Function RunApp($path, $message, $params) {

```

Comment:
args is a keyword for passing unnamed args to a powershell script, so renamed it to "params" for our usage here.